File: js/reverse-words.js
/**
 * Tanggal: 17/3/25
 * Link: https://www.codewars.com/kata/5259b20d6021e9e14c0010d4/train/javascript
 */

function reverseWords(str) {
    return str
        .split(" ")
        .map((word) => word.split("").reverse().join(""))
        .join(" ");
}

console.log(reverseWords("The quick brown fox jumps over the lazy dog."));

File: python/reverse-words.py
# Tanggal: 12 Mei 2026
# Link: https://www.codewars.com/kata/57a55c8b72292d057b000594/train/python

def reverse_words(s):
    result = s.split()
    result.reverse()
    return " ".join(result)

if __name__ == "__main__":
    print(reverse_words("Hello world"))
